content/docs/fragments.md

```diff
@@ -38,93 +37,93 @@ class Table extends React.Component {
 }
 ```
 
-`<Columns />` would need to return multiple `<td>` elements in order for the rendered HTML to be valid. If a parent div was used inside the `render()` of `<Columns />`, then the resulting HTML will be invalid.
+`<Columns />` tendría que devolver múltiples elementos `<td>` para que el código HTML renderizado sea válido. Si un div padre fue utilizado dentro del código `render()` de `<Columns />`, entonces el código HTML resultante será inválido.
 
 ```jsx
 class Columns extends React.Component {
   render() {
     return (
       <div>
-        <td>Hello</td>
-        <td>World</td>
+        <td>Hola</td>
```

Comment:
Hey @RamirezAlex, we're leaving the code as is, we're only translating comments. So please, leave this as `<td>Hello</td>`

We use lots of codepen examples, and if we translate the code contents we could confuse people!